Logos/hpe_aruba_r_pos_rgb.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-5840 1394.6 792 612" style="enable-background:new -5840 1394.6 792 612;" xml:space="preserve">

```

Comment:
In svg id should be guid and it should not have style tag and xmlns:xlink. The size should not more then 5kb.

Please follow the guideline mention below:
Validator Rules:

proper formatting
size is under 5k
illegal attributes: data-name, style, xmlns:link
Do not use xink:href - use inline instead
Do not use title tags, as they cannot be localized
Each icon needs to be exported with its Object IDs setting set to Unique. If it’s not, Illustrator will generate a generic ID for each layer in the SVG code. This is problematic because if there is a matching ID anywhere else in product code, the icon will be overwritten and not display

You can also refer the guideline [here
](https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md)

Thanks,
Aditi Maheshwari

Reply:
@rkent-hpe In svg id should be guid and it should not have style tag and xmlns:xlink. The size should not more then 5kb.

Please follow the guideline mention below:
Validator Rules:

proper formatting
size is under 5k
illegal attributes: data-name, style, xmlns:link
Do not use xink:href - use inline instead
Do not use title tags, as they cannot be localized
Each icon needs to be exported with its Object IDs setting set to Unique. If it’s not, Illustrator will generate a generic ID for each layer in the SVG code. This is problematic because if there is a matching ID anywhere else in product code, the icon will be overwritten and not display

You can also refer the guideline [here](https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md)

Thanks,
Aditi Maheshwari